Dockerfile

```diff
@@ -9,6 +9,9 @@ ENV ELABIMG_VERSION 2.4.0
 
 ENV S6_OVERLAY_VERSION 2.2.0.1
 
+# the architecture to build for, necessary for S6
+ENV ARCHITECTURE=$arch
```

Comment:
but then this wouldn't work for my docker hub builds, there is a need for hooks to override the build command: https://docs.docker.com/docker-hub/builds/advanced/#override-build-test-or-push-commands

Too bad we can't have default env value to avoid a breaking change!

Reply:
Yes this is unfortunate. But then should it not be possible if I add such hooks or would this also impact your images? Basically I want images to autobuild when there are new versions of elab... more complicated than I thought